packages/doc-spa/src/pages/markdown-page.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Is this page required? This looks like a placeholder from the docusaurus generator.